src/main/java/com/wenqi/bootwebdemo/controller/PersonController.java

```diff
@@ -0,0 +1,42 @@
+package com.wenqi.bootwebdemo.controller;
+
+import com.wenqi.bootwebdemo.exception.PersonNotFoundException;
+import com.wenqi.bootwebdemo.model.Person;
+import com.wenqi.bootwebdemo.model.PersonResponseDTO;
+import com.wenqi.bootwebdemo.service.PersonService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class PersonController {
+
+    @Autowired
+    PersonService personService;
+
+    @GetMapping(path = "/persons", produces = {"application/json"})
+    public List<Person> getPersonList() {
+        List<Person> personList = personService.getAllPersons();
+        return personList;
+    }
+
+    @GetMapping("/person/{aid}")
+    public String getPersonById(@PathVariable("aid") int aid) {
```

Comment:
myself generally prefers Integer over over, but Integer can be null to handle empty state but int defaults to 0